javascript/javascript3/week2/excercise.js

```diff
@@ -0,0 +1,80 @@
+//1-movies
+const moviesApi =
+  "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json";
+fetch(moviesApi)
+  .then((response) => response.json())
+  .then((data) => {
+    const badMovies = data.filter((movie) => movie.rating < 5);
+    const badMovies2000 = badMovies
+      .filter((movie) => movie.year >= 2000)
+      .map((movie) => movie.title);
+    console.log(badMovies);
+    console.log(badMovies2000);
+  })
+  .catch((err) => console.log(err.message));
+//2-Promise that resolves after set time
+function resolveTimeout(resolveAfter) {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      resolve();
+    }, resolveAfter * 3000);
+  });
+}
+resolveTimeout(8).then(() => {
+  console.log("I am called asynchronously.");
+});
+
+async function resolveTimeout2(resolveAfter) {
```

Comment:
Function name could be changed to more logical name like resolveTimeoutWithAsync() instead on resolveTimeout2.

Reply:
Thanks for your suggestion. I will change it